valhalla/sif/edgelabel.h

```diff
@@ -411,7 +424,7 @@ class EdgeLabel {
   uint64_t deadend_ : 1;
   uint64_t on_complex_rest_ : 1;
   uint64_t closure_pruning_ : 1;
-  uint64_t spare_0 : 1;
+  uint64_t has_flow_speed_ : 1;
 
```

Comment:
Our new bit for flow.

Reply:
i would suggest we name that `has_measured_speed_` since we really want this for any edge that has measured traffic speed (flow/constrained/historical/realtime)

Reply:
@kevinkreiser updated

Reply:
no more spare bits left :slightly_frowning_face:

Reply:
there are 17 more spare below. there just arent anymore spare in _this_ 64bit value